src/GRA.Data/Repository/TriggerRepository.cs

```diff
@@ -144,48 +144,46 @@ private IQueryable<Model.Trigger> ApplyFilters(BaseFilter filter)
             // create a list of triggers to remove based on badge and challenge earnings
             var itemsToRemove = new List<Model.Trigger>();
 
-            // get a list of triggers that fire based on badge earnings
-            var badgeTriggers = triggers
-                .Where(_ => _.RequiredBadges != null && _.RequiredBadges.Count > 0);
+            // get a list of triggers that fire based on badge or challenge earnings
+            var itemTriggers = triggers.Where(_ => _.ItemsRequired > 0);
```

Comment:
So this means that in order for badges and challenges to work from here on forward the `ItemsRequired` property will need to be set, am I reading that right?